react-ui/src/components/SignUpForm.js

```diff
@@ -68,22 +73,47 @@ render() {
                 </label>
                 <br></br>
                 <br></br>
-                <label for="role" className ="Title-Style2"> Select Role: </label>
-                <select name="role" className ="InputFields2" id="type">
+                <label for="role" className ="Title-Style2"> Select User Role: </label>
+                <select name="role" className ="InputFields3" id="type">
                   <option value="student">Student</option>
+                  <option value="worker1">Worker</option>
+                </select>
+                <br></br>
+                <br></br>
+                <label for="worker_type" className ="Title-Style2"> Select Worker Type (or N/A otherwise): </label>
```

Comment:
Created a dropdown as per design specification document for service workers to select option from drop down menu or select "other" and input their designation in the appropriate input box below. Students are expected to select the N/A option from this dropdown.